bert_reranker/main.py

```diff
@@ -66,8 +66,18 @@ def main():
     tokenizer_name = hyper_params['tokenizer_name']
     tokenizer = AutoTokenizer.from_pretrained(tokenizer_name)
 
-    train_dataloader, dev_dataloader = generate_dataloaders(
-        hyper_params['train_file'], hyper_params['dev_file'], hyper_params['cache_folder'],
+    train_dataloader = generate_dataloader(
+        hyper_params['train_file'], hyper_params['cache_folder'],
+        hyper_params['max_question_len'], hyper_params['max_paragraph_len'],
+        tokenizer, hyper_params['batch_size'])
+
+    dev_dataloader = generate_dataloader(
```

Comment:
@jerpint would it be possible to have this supporting any dev set? Like, in the config yaml you can specify a list of files under dev

dev_files:
  - 'file1.json'
  - 'file2.json'
 - ...

Reply:
good idea!